pkg/features/kube_features.go

```diff
@@ -343,6 +343,12 @@ const (
 	//
 	// Enables CSI to use raw block storage volumes
 	CSIBlockVolume utilfeature.Feature = "CSIBlockVolume"
+
+	// owner: @yastij
+	// alpha: v1.12
+	//
+	// Enables detaching volumes when a node reports nodeShutdown taint
+	NodeShutdown utilfeature.Feature = "nodeShutdown"
```

Comment:
Please add this feature to the [SIG-storage tracking spreadsheet](https://docs.google.com/spreadsheets/d/1t4z5DYKjX2ZDlkTpCnp18icRAQqOE85C1T1r2gqJVck/edit#gid=1099909957) and, if possible, attend the SIG-storage meetings to provide regular updates on it.

Reply:
Sure, thanks for pointing this.